deb/deb.go

```diff
@@ -74,6 +75,13 @@ func GetControlFileFromDeb(packageFile string) (Stanza, error) {
 				}
 				defer unxz.Close()
 				tarInput = unxz
+			case "control.tar.zst":
+				unzstd, err := zstd.NewReader(bufReader)
+				if err != nil {
+					return nil, errors.Wrapf(err, "unable to unzstd data.tar.zstd from %s", packageFile)
```

Comment:
I think instead of `data.tar.zstd` here you want `%s` with the `header.Name` like the other compression options (or just `control.tar.zst`, they should be the same).

Reply:
```suggestion
					return nil, errors.Wrapf(err, "unable to unzstd %s from %s", header.Name, packageFile)
```

Reply:
oh thanks for catching the string typo, its actually done slightly differently for the data decompress at L180+ which is where i compared, ill tidy it up now